ocaml/doc/wire-protocol.md

```diff
@@ -170,7 +170,7 @@ to subequent RPC calls as an authentication token.
 A session can be terminated with the `session.logout` function:
 
 ```python
-   void  session.logout(session_id session)
+   void  session.logout(session ref)
```

Comment:
Shouldn't this be `session ref session_id` to be consistent with the `type name` structure used above?